resources/js/components/BackupTool.vue

```diff
@@ -131,15 +133,21 @@ export default {
         startPolling() {
             if(Nova.config.nova_backup_tool.polling){
                 const poller = window.setInterval(() => {
-                    this.updateBackupStatuses();
-                    this.updateActiveDiskBackups();
+                    if (!this.showingModal) {
+                        this.updateBackupStatuses();
+                        this.updateActiveDiskBackups();
+                    }
                 }, Nova.config.nova_backup_tool.polling_interval * 1000);
 
                 this.$once('hook:beforeDestroy', () => {
                     window.clearInterval(poller);
                 });
             }
         },
+
+        toggleModal({ state }) {
```

Comment:
I think `setModalVisibility` would be a better now, as you're not really toggling.